index.html

```diff
@@ -71,7 +72,220 @@ <h1>I’m Sylvester,
+      </div>
+      <div class="card-head">
+        <h1>Multi-Post <br />Stories</h1>
+      </div>
+      <div class="card-info">
+        <span class="canopy">CANOPY</span>
+        <i class="fas fa-circle fa-xs"></i>
+        <span class="head-item">Back End Dev</span>
+        <i class="fas fa-circle fa-xs"></i>
+        <span class="head-item">2015</span>
+      </div>
+      <div class="card-para">
+        <p>
+          A daily selection of privately personalized reads; no accounts
+          or sign-ups required.
+        </p>
+      </div>
+      <div class="card-lang">
+        <ul>
+          <li class="html">html</li>
+          <li class="css">css</li>
+          <li class="js">javaScript</li>
+        </ul>
+      </div>
+      <div class="seeport">
+        <span>
+          <a href="#">See Project</a>
+        </span>
+      </div>
+    </section>
+    <section class="about">
+      <div class="about-me">
+        <h1>About <br /> Myself</h1>
+      </div>
+      <div class="about-text">
+        <p>
+          Hello I’m a software developer! I can help you build a product, 
+          feature or website Look through some of my work and experience! 
+          If you like what you see and have a project you need coded, 
+          don’t hestiate to contact me.
+        </p>        
+      </div>
+      <a href="#" class="connect2">LET'S CONNECT</a>
+      <div class="social-icons2">
+        <ul>
+          <li>
+            <a href="#">
+              <img src="images/twitter.png" alt="Connect on Twitter">
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="images/I.png" alt="Connect on Twitter">
+              <span>
+                <img src="images/n.png" alt="">
+              </span>
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="images/connect-m.png" alt="Connect on IM">
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="images/github.png" alt="Connect on GitHub">
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="images/thumbs-up.png" alt="Two fingers">
+            </a>
+          </li>
+        </ul>
+      </div>
+      <div class="getresume">
+        <span>
+          <a href="#">Get my resume</a>
+        </span>
+      </div>
+    </section>
+    <section class="language">
+      <div class="language-head">
+        <span>Languages</span>
+        <span><i class="fas fa-chevron-down"></i></span>
+      </div>
+      <div class="java">
+        <img src="images/javascript.png" alt="Javascript" />
+        <span>JavaScript</span>
+      </div>
+      <div class="java">
+        <img src="/portfolio-project/images/html.png" alt="HTML" />
+        <span>HTML</span>
+      </div>
+      <div class="java">
+        <img src="images/css.png" alt="CSS" />
+        <span>CSS</span>
+      </div>
```

Comment:
- [ ] The list of tools under the `language` category should be defined in `li` tag and wrap them with a `ul` tag.

Reply:
The issue is addressed. Thankyou! :+1: